.github/workflows/ci.yml

```diff
@@ -2,9 +2,9 @@ name: Continuous Integration
 
 on:
   push:
-    branches: [master, staging, trying]
```

Comment:
Why does this need to be removed? I think the `pull_request` will trigger here.

Reply:
@nkrackow Forgot about this when approving. Could you comment and address in some later PR?

Reply:
I wasn't sure how I get it to run CI on pushes to any branch. I thought I had it like this for thermostat and it ran on pushes to any branch but I did not check this.